### ISSUE TITLE
processCss ignores props passed to cssnano via loader query

### ISSUE THREAD

**Issue by mdreizin** [webpack/css-loader#150]
`css-loader?{"sourceMap":true,"minimize":true,"safe":true}`

[lib/processCss.js:157](https://github.com/webpack/css-loader/blob/master/lib/processCss.js#L157) should use `Object.assign` instead of `Object.create`.

```js
var minimizeOptions = Object.create(query);
```
`minimizeOptions` will be:
```json
{}
```

```js
var minimizeOptions = Object.assign({}, query);
```
`minimizeOptions` will be:
```json
{ "sourceMap": true, "minimize": true, "safe": true } 
```

cc @sokra

**Comment by jbmusso** [comment]
Hi, thanks for spotting this. I'm currently facing the same issue when trying to circumvent the default `webpack -p` behavior. Indeed, `webpack -p` seems to be minifying the CSS in a destructive way by default because of cssnano default behavior (to the best of my understanding).
Related: a default behavior of the `webpack -p` flag could be that webpack unsures a non-destructive minification of css (default to cssnano safe mode, with aggressive/destructive mode being optional/explicit, maybe via a `--optimize-aggressive` flag). Maybe this could be better raised as a separate issue on the webpack repository.

A `safe` option was indeed added in css-nano (see https://github.com/ben-eb/cssnano/pull/59 merged and available in v3.2.0).

Versions:
- webpack v1.12.2
- css-loader v0.19.0 with cssnano v3.2.0

**Comment by salzhrani** [comment]
I confirm that @mdreizin pr fixes this issue
